clouddriver-api/clouddriver-api.gradle

```diff
@@ -23,8 +23,15 @@ dependencies {
   api("io.spinnaker.kork:kork-plugins-api")
   api("io.spinnaker.kork:kork-credentials-api")
 
+  implementation "com.google.guava:guava"
```

Comment:
Ideally none of these dependencies would be added ☹️ 

The reasoning behind limiting dependencies is that a plugin developer should be able to depend on the interfaces in the `clouddriver-api` package until there's a breaking change in the interface. If this package brings its own version of Spring, then the plugin will stop working whenever that version conflicts with the version of Guava inside Clouddriver. Many of the api packages have big warnings in them: https://github.com/spinnaker/echo/blob/master/echo-api/echo-api.gradle#L1-L7